app/pages/profile/page.tsx

```diff
@@ -41,35 +105,58 @@ export default function Profile() {
           <LogoutButton />
         </div>
       </div>
-      {courses.length !== 0 ? (
-        <div className="mt-12 text-left ml-12">
-          <p className="mb-5 font-bold text-2xl">Courses Ratings</p>
-          {courses.map((c) => (
-            <div
-              key={c.courses.id}
-              className="flex flex-row justify-between mt-4 mb-4 rounded-md py-5 px-5 w-1/2 border-2 border-black"
-            >
-              <p className="pr-6 font-size text-xl">
-                {c.courses.course_code} - {c.courses.title}
-              </p>
-              <div className="flex">
-                {[...Array(5)].map((star, index) => {
-                  const id = index;
-                  const ratingValue = index + 1;
-                  return (
-                    <FaStar
-                      key={id}
-                      className="star"
-                      color={ratingValue <= (c.rating ?? 0) ? '#ffc107' : '#e4e5e9'}
-                      size={30}
-                    />
-                  );
-                })}
-              </div>
+      <div className="flex flex-row">
+        <div className="flex flex-col w-1/2 mr-10">
+          {courses.length !== 0 ? (
```

Comment:
This boolean condition can be simplified, right? An empty array is falsy.

```suggestion
          {courses ? (
```

Reply:
Its easier to understand the meaning of the code using length, courses ? can be thought of as if it's null.